libs/SalesforceReact/build.gradle.kts

```diff
@@ -137,12 +137,9 @@ task<Exec>("buildReactTestBundle") {
 }
 
 task("buildReactTestBundleIfNotExists") {
-    doLast {
```

Comment:
This `doLast` specifically fails in Gradle/AGP 8.  Everything does seem to test correctly without it, however.  I'd be game to know more if anyone recalls why this was so.